duckduckgo/Android
# ADS: Custom Builders for AlertDialog components

Task/Issue URL: https://app.asana.com/0/0/1203198140265639/f

### Description

- Create custom builders for each dialog component: `TextAlertDialog`, `StackedAlertDialog` and `RadioListAlertDialog`
- Moved dialog background color to style from rootView

### Steps to test this PR

- Install from this branch
- Go to Settings > App Components Design Preview
- Tap on "Dialogs" tab
- [ ] Check all dialogs under DIALOGS title look fine

### No UI changes
